workshop/exercise-1/1-questions.md

```diff
@@ -5,94 +5,101 @@
 
 Type true/false in the provided [ ].
 
-a) [ ] `<div><span>hello</div></span>`
+a) [true] `<div><span>hello</div></span>`
```

Comment:
You might want to check this answer over.

Reply:
Thanks